src/pfs/pfs.go

```diff
@@ -69,6 +69,7 @@ func (r *Repo) NewBranch(name string) *Branch {
 func (r *Repo) NewCommit(branch, id string) *Commit {
```

Comment:
Should a new branchless method replace this? You could use deprecation to encourage users to change to `.NewProvenantCommit` or whatever name you pick.